src/cpp/r/include/r/session/RSession.hpp

```diff
@@ -57,7 +57,7 @@ struct ROptions
 {
    ROptions() :
          useInternet2(true),
-         rCompatibleGraphicsEngineVersion(9),
+         rCompatibleGraphicsEngineVersion(12),
```

Comment:
It looks like we set the compatible graphics engine version in a bunch of places, so we should sanity check that these changes make sense. (I'm a little confused because the versions declared in the various places are different)